java/SQSSink/src/main/java/com/amazonaws/services/msf/SQSStreamingJob.java

```diff
@@ -12,6 +12,7 @@
 import org.apache.flink.streaming.api.datastream.DataStream;
 import org.apache.flink.streaming.api.environment.LocalStreamEnvironment;
 import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+import org.apache.flink.formats.json.JsonSerializationSchema;
```

Comment:
I think `flink-json` depemdency is required for this class (`scope` = `provided`)